test/e2e/suite_test.go

```diff
@@ -1668,3 +1669,20 @@ func checkInjectedContainersPresence(t *testing.T, ctx context.Context, f *frame
 	}
 	return nil
 }
+
+func checkInjectedVolumePresence(t *testing.T, ctx context.Context, f *framework.E2eFramework, dcKey framework.ClusterKey) error {
+	t.Logf("check that volumes were injected in %s cass pods in cluster %s", dcKey.Name, dcKey.K8sContext)
+	cassdc := &cassdcapi.CassandraDatacenter{}
+	err := f.Get(ctx, dcKey, cassdc)
+	if err != nil {
+		return err
+	}
+
+	require.Equal(t, 1, len(cassdc.Spec.StorageConfig.AdditionalVolumes), "expected 1 additional volume")
```

Comment:
In the containers PR, you performed verification of what we expect from cass-operator. We should do the same here.

Reply:
Done 👍